networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -16,7 +16,7 @@ For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws
 [id="aws-load-balancer-operator-release-notes-1.0.0"]
 == AWS Load Balancer Operator 1.0.0
 
-The AWS Load Balancer Operator is now generally available with this release.
+The AWS Load Balancer Operator is now generally available with this release. The AWS Load Balancer Operator version 1.0.0 supports the AWS Load Balancer Controller version 2.4.
```

Comment:
Just noticed that the version of the controller is 2 digits while the operator is 3. To be precise we use the version `2.4.4` for `1.0.0` ALBO.